Lib/annotationlib.py

```diff
@@ -178,6 +178,11 @@ def evaluate(
 
         arg = self.__forward_arg__
         if arg.isidentifier() and not keyword.iskeyword(arg):
+            if self.__forward_module__ is not None:
```

Comment:
I don't think this is correct, since we infer globals from `__forward_module__` if no explicit globals are given. And if explicit globals are given, we shouldn't be using `__forward_module__`.

Reply:
I don't think current behavior is correct. Was there any discussion?